pom.xml

```diff
@@ -120,6 +120,41 @@
             <artifactId>tokens</artifactId>
             <version>0.11.0-beta-2</version>
         </dependency>
+        <dependency>
+            <groupId>io.opentracing.contrib</groupId>
+            <artifactId>opentracing-spring-web-autoconfigure</artifactId>
```

Comment:
Have you compared this dependency to https://github.com/opentracing-contrib/java-spring-cloud ?
The initialisation of the bean is the same, but looks like latter provides some extras.